host_vars/dev-handlers.usegalaxy.org.au.yml

```diff
@@ -156,6 +156,13 @@ host_galaxy_config: # renamed from __galaxy_config
     # TUS
     tus_upload_store: "{{ galaxy_tus_upload_store }}"
     datatypes_config_file: "{{ galaxy_server_dir }}/lib/galaxy/config/sample/datatypes_conf.xml.sample,{{ galaxy_config_dir }}/additional_datatypes_conf.xml"
+    smtp_server: localhost
+    error_email_to: <help@genome.edu.au> # error reports are disabled if no email is set
+    email_from: <galaxy-no-reply@dev.usegalaxy.org.au>
+    blacklist_file: "{{galaxy_config_dir }}/email_domain_blocklist.conf"
```

Comment:
is this `email_domain_blocklist_file`? https://github.com/galaxyproject/galaxy/blob/release_22.05/lib/galaxy/config/sample/galaxy.yml.sample#L909

Reply:
I can't find that blocklist.conf anywhere

Reply:
The blocklist.conf is in files. It's one of the files listed in the Files Changed section about (it's large and so the contents are not shown by default)

Reply:
And it's still called blacklist in the documentation. :( I'll change it to `email_domain_blocklist_file:`